lib/utils.js

```diff
@@ -1,5 +1,5 @@
 // NPM dependencies
-const getKeypath = require('keypather/get');
+const { get: getKeypath } = require('lodash');
```

Comment:
this is assigning to `get` but the code later still refers to `getKeyPath`

Reply:
Must admit I was unfamiliar with this syntax, but in the previous change on the govuk kit (see https://github.com/alphagov/govuk-prototype-kit/pull/1155/files#diff-5dfe38baf287dcf756a17c2dd63483781b53bf4b669e10efdd01e74bcd8e780aR38) the `getKeypath` line was also left unchanged, so I assumed it was importing the `get` function from lodash but assigning it to `getKeyPath`?

Reply:
ah yes, looks like it's using this syntax to destructure the `get` function from lodash and assign it to the `getKeypath` local variable? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#assigning_to_new_variable_names

Reply:
gosh thats confusing syntax!

Reply:
heh, yes! Probably this could be refactored to be less confusing, but it felt slightly safer to stick to the exact changes made on the GOVUK prototype kit for now.